unittests/test_parsing.py

```diff
@@ -22,54 +21,56 @@ def test_find_file_to_format(self):
+            )
             assert f"No file found for {EdifactFormat.ORDRSP}." in caplog.text
-            assert file_dict[EdifactFormat.ORDCHG] == input_dir / Path("ORDCHG_MIG_1_1_info_20230331_v2.docx")
+            assert (
+                file_dict[EdifactFormat.ORDCHG]
+                == path_to_test_FV2310 / "ORDCHGMIG-informatorischeLesefassung1.1_99991231_20231001.docx"
+            )
 
     def test_find_only_one_file_multiple_docx(self):
         """
         Tests to find multiple docx files with the same message_format.
         """
         message_formats = [EdifactFormat.IFTSTA]
-        input_dir = Path("unittests/test_data/")
-        file_dict = find_file_to_format(message_formats, input_dir)
-        assert file_dict[EdifactFormat.IFTSTA] == input_dir / Path(
-            "IFTSTAMIG-informatorischeLesefassung2.0emitFehlerkorrekturenStand11.03.2024_99991231_20240311.docx"
+        file_dict = find_file_to_format(
+            message_formats, path_to_test_edi_energy_mirror_repo, EdifactFormatVersion.FV2310
+        )
+        assert (
+            file_dict[EdifactFormat.IFTSTA]
+            == path_to_test_FV2310
+            / "IFTSTAMIG-informatorischeLesefassung2.0emitFehlerkorrekturenStand11.03.2024_99991231_20240311.docx"
         )
 
     def test_parse_raw_nachrichtenstrukturzeile(self):
         """
         Test to parse the raw nachrichtenstrukturzeile from a docx file.
         """
-        input_file = Path("unittests/test_data/ORDCHG_MIG_1_1_info_20230331_v2.docx")
+        input_file = path_to_test_FV2310 / "ORDCHGMIG-informatorischeLesefassung1.1_99991231_20231001.docx"
         mig_table = parse_raw_nachrichtenstrukturzeile(input_file)
-        assert len(mig_table) == 18
-        assert "Nachrichten-Kopfsegment" in mig_table[0]
-        assert "Nachrichten-Endesegment" in mig_table[-1]
-
-    def test_preliminary_output_as_json(self, tmp_path):
-        """Tests the preliminary output as json function.
-        Asserts that the outputfile exists and has the correct content."""
-        table = ["line1", "line2", "line3"]
-        message_format = EdifactFormat.ORDCHG
-        output_dir = tmp_path / Path("output")
-
-        preliminary_output_as_json(table, message_format, output_dir)
-
-        file_path = output_dir / f"{message_format}_preliminary_output.json"
-        assert file_path.exists()
-
-        with open(file_path, "r", encoding="utf-8") as json_file:
-            content = json.load(json_file)
-            assert content == {"line1": None, "line2": None, "line3": None}
+        expected_csv_file = (
+            expected_output_dir / EdifactFormatVersion.FV2310 / EdifactFormat.ORDCHG / "nachrichtenstruktur.csv"
+        )
+        with open(expected_csv_file, encoding="utf-8") as csvfile:
+            number_of_lines_in_csv = sum(1 for _ in csvfile) - 1  # excl. header
+            assert len(mig_table) == number_of_lines_in_csv
+            csvfile.seek(0)
```

Comment:
What does this command?

Reply:
This first ensures that the actual table and the expected table have the same number of row (excluding the header in the csv file). We then jump back to the beginning of the csv file and start comparing line by line.